widgets/common/src/filters/RuleRow.tsx

```diff
@@ -1,16 +1,16 @@
-import type { FunctionComponent } from 'react';
+import type { ComponentProps, FunctionComponent } from 'react';
 import RuleAttributeDropdown from './RuleAttributeDropdown';
 import RuleOperatorDropdown from './RuleOperatorDropdown';
 import RuleInput from './RuleInput';
 import RuleRemoveButton from './RuleRemoveButton';
-import { FormField, FormGroup } from './FormWrapper';
 
 interface RuleRowProps {
-    id: number;
-    onRemove: (id: number) => void;
+    allowRemove: boolean;
```

Comment:
Optional:

Perhaps 

```suggestion
    removable: boolean;
```

would be a bit shorter :smile:

Reply:
Changed.